src/handlers/comment/handlers/first.ts

```diff
@@ -28,9 +31,17 @@ export const verifyFirstCheck = async (): Promise<void> => {
         per_page: 100,
       });
       const isFirstComment = resp.data.filter((item) => item.user?.login === payload.sender.login).length === 1;
-      if (isFirstComment) {
+      if (isFirstComment && enabled) {
```

Comment:
this var could've been refactored to something else, "enabled" sounds global in many context

Reply:
Well this is in a handler specific for this feature so I think it's probably fine.